# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Fix unknown symbols in collation rules.  These symbols
# DP:    are trapped by locale/check-unknown-symbols.diff
# DP: Dpatch author: Denis Barbier <barbier@debian.org>
# DP: Patch author: Denis Barbier
# DP: Upstream status: not submitted
# DP: Date: 2006-01-19

Index: glibc-2.3.5/localedata/locales/da_DK
===================================================================
--- glibc-2.3.5.orig/localedata/locales/da_DK
+++ glibc-2.3.5/localedata/locales/da_DK
@@ -66,6 +66,7 @@
 collating-symbol <ae>
 collating-symbol <o-stroke>
 collating-symbol <a-ring>
+collating-symbol <ETH>
 
 reorder-after <RES-1>
 <CAP>
Index: glibc-2.3.5/localedata/locales/fa_IR
===================================================================
--- glibc-2.3.5.orig/localedata/locales/fa_IR
+++ glibc-2.3.5/localedata/locales/fa_IR
@@ -97,6 +97,7 @@
 collating-symbol <ALT2>
 
 collating-element <Alef-Madda> from "<U0627><U0653>"
+collating-element <Alef-Hamza> from "<U0627><U0654>"
 collating-element <Alef-HamzaBelow> from "<U0627><U0655>"
 collating-element <Waw-Hamza> from "<U0648><U0654>"
 collating-element <AlefMaksura-Hamza> from "<U0649><U0654>"
Index: glibc-2.3.5/localedata/locales/is_IS
===================================================================
--- glibc-2.3.5.orig/localedata/locales/is_IS
+++ glibc-2.3.5/localedata/locales/is_IS
@@ -947,7 +947,6 @@
 <UE00D>	IGNORE;IGNORE;IGNORE;<UE00D>
 <UE00E>	IGNORE;IGNORE;IGNORE;<UE00E>
 <UE00F>	IGNORE;IGNORE;IGNORE;<UE00F>
-<"=>	IGNORE;IGNORE;IGNORE;<"=>
 <UE011>	IGNORE;IGNORE;IGNORE;<UE011>
 <UE012>	IGNORE;IGNORE;IGNORE;<UE012>
 <UE013>	IGNORE;IGNORE;IGNORE;<UE013>
@@ -956,8 +955,6 @@
 <UE017>	IGNORE;IGNORE;IGNORE;<UE017>
 <UE018>	IGNORE;IGNORE;IGNORE;<UE018>
 <UE019>	IGNORE;IGNORE;IGNORE;<UE019>
-<Mc>	IGNORE;IGNORE;IGNORE;<Mc>
-<Fl>	IGNORE;IGNORE;IGNORE;<Fl>
 UNDEFINED	IGNORE;IGNORE;IGNORE
 
 <U0030>	<U0030>;<U0030>;IGNORE;IGNORE
Index: glibc-2.3.5/localedata/locales/lo_LA
===================================================================
--- glibc-2.3.5.orig/localedata/locales/lo_LA
+++ glibc-2.3.5/localedata/locales/lo_LA
@@ -230,6 +230,7 @@
 collating-element <ho-ai-maimuan> from "<U0EC3><U0EAB>"
 collating-element <ho-ai-maimalai> from "<U0EC4><U0EAB>"
 
+collating-element <o-e> from "<U0EC0><U0EAD>"
 collating-element <o-ae> from "<U0EC1><U0EAD>"
 collating-element <o-o> from "<U0EC2><U0EAD>"
 collating-element <o-ai-maimuan> from "<U0EC3><U0EAD>"
@@ -601,10 +602,10 @@
 
 <U0EDD>        <U0EDC>;<BLANK>;<BLANK>;<BLANK> % Lao CHARACTER HHOr
 <mhor-e>           "<U0EDD><U0EC0>";<BLANK>;<BLANK>;<BLANK>
-<mor-ae>          "<U0EDD><U0EC1>";<BLANK>;<BLANK>;<BLANK>
-<mor-o>           "<U0EDD><U0EC2>";<BLANK>;<BLANK>;<BLANK>
-<mor-ai-maimuan>  "<U0EDD><U0EC3>";<BLANK>;<BLANK>;<BLANK>
-<mor-ai-maimalai> "<U0EDD><U0EC4>";<BLANK>;<BLANK>;<BLANK>
+<mhor-ae>          "<U0EDD><U0EC1>";<BLANK>;<BLANK>;<BLANK>
+<mhor-o>           "<U0EDD><U0EC2>";<BLANK>;<BLANK>;<BLANK>
+<mhor-ai-maimuan>  "<U0EDD><U0EC3>";<BLANK>;<BLANK>;<BLANK>
+<mhor-ai-maimalai> "<U0EDD><U0EC4>";<BLANK>;<BLANK>;<BLANK>
 
 
 % order of Lao vowels
Index: glibc-2.3.5/localedata/locales/lv_LV
===================================================================
--- glibc-2.3.5.orig/localedata/locales/lv_LV
+++ glibc-2.3.5/localedata/locales/lv_LV
@@ -919,7 +919,6 @@
 <UE00D>	IGNORE;IGNORE;IGNORE;<UE00D>
 <UE00E>	IGNORE;IGNORE;IGNORE;<UE00E>
 <UE00F>	IGNORE;IGNORE;IGNORE;<UE00F>
-<"=>	IGNORE;IGNORE;IGNORE;<"=>
 <UE011>	IGNORE;IGNORE;IGNORE;<UE011>
 <UE012>	IGNORE;IGNORE;IGNORE;<UE012>
 <UE013>	IGNORE;IGNORE;IGNORE;<UE013>
Index: glibc-2.3.5/localedata/locales/nb_NO
===================================================================
--- glibc-2.3.5.orig/localedata/locales/nb_NO
+++ glibc-2.3.5/localedata/locales/nb_NO
@@ -59,6 +59,7 @@
 collating-symbol <ae>
 collating-symbol <o-stroke>
 collating-symbol <a-ring>
+collating-symbol <ETH>
 
 reorder-after <RES-1>
 <CAP>
Index: glibc-2.3.5/localedata/locales/sr_CS
===================================================================
--- glibc-2.3.5.orig/localedata/locales/sr_CS
+++ glibc-2.3.5/localedata/locales/sr_CS
@@ -137,6 +137,21 @@
 
 copy "iso14651_t1"
 
+collating-element <D-Z<> from "<U0044><U017D>"
+collating-element <D-z<> from "<U0044><U017E>"
+collating-element <d-Z<> from "<U0064><U017D>"
+collating-element <d-z<> from "<U0064><U017E>"
+
+collating-element <L-J> from "<U004C><U004A>"
+collating-element <L-j> from "<U004C><U006A>"
+collating-element <l-J> from "<U006C><U004A>"
+collating-element <l-j> from "<U006C><U006A>"
+
+collating-element <N-J> from "<U004E><U004A>"
+collating-element <N-j> from "<U004E><U006A>"
+collating-element <n-J> from "<U006E><U004A>"
+collating-element <n-j> from "<U006E><U006A>"
+
 collating-symbol  <d-z-sh>
 collating-symbol  <l-j-sh>
 collating-symbol  <n-j-sh>
@@ -179,35 +194,35 @@
 
 reorder-after <U0064>
 <d-z<>	<d-z-sh>;<PCT>;<MIN>;IGNORE
-<dz<>	<d-z-sh>;<PCT>;<MIN>;IGNORE
+<U01C6>	<d-z-sh>;<PCT>;<MIN>;IGNORE
 <d-Z<>	<d-z-sh>;<PCT>;<MIN-CAP>;IGNORE
 <U0111>	<d//>;<PCT>;<MIN>;IGNORE
 reorder-after <U0044>
 <D-Z<>	<d-z-sh>;<PCT>;<CAP>;IGNORE
-<DZ<>	<d-z-sh>;<PCT>;<CAP>;IGNORE
+<U01C4>	<d-z-sh>;<PCT>;<CAP>;IGNORE
 <D-z<>	<d-z-sh>;<PCT>;<CAP-MIN>;IGNORE
-<Dz<>	<d-z-sh>;<PCT>;<CAP-MIN>;IGNORE
+<U01C5>	<d-z-sh>;<PCT>;<CAP-MIN>;IGNORE
 <U0110>	<d//>;<PCT>;<CAP>;IGNORE
 
 reorder-after <U006C>
 <l-j>	<l-j-sh>;<PCT>;<MIN>;IGNORE
-<lj>	<l-j-sh>;<PCT>;<MIN>;IGNORE
+<U01C9>	<l-j-sh>;<PCT>;<MIN>;IGNORE
 <l-J>	<l-j-sh>;<PCT>;<MIN-CAP>;IGNORE
 reorder-after <U004C>
 <L-J>	<l-j-sh>;<PCT>;<CAP>;IGNORE
-<LJ>	<l-j-sh>;<PCT>;<CAP>;IGNORE
+<U01C7>	<l-j-sh>;<PCT>;<CAP>;IGNORE
 <L-j>	<l-j-sh>;<PCT>;<CAP-MIN>;IGNORE
-<Lj>	<l-j-sh>;<PCT>;<CAP-MIN>;IGNORE
+<U01C8>	<l-j-sh>;<PCT>;<CAP-MIN>;IGNORE
 
 reorder-after <U006E>
 <n-j>	<n-j-sh>;<PCT>;<MIN>;IGNORE
-<nj>	<n-j-sh>;<PCT>;<MIN>;IGNORE
+<U01CC>	<n-j-sh>;<PCT>;<MIN>;IGNORE
 <n-J>	<n-j-sh>;<PCT>;<MIN-CAP>;IGNORE
 reorder-after <U004E>
 <N-J>	<n-j-sh>;<PCT>;<CAP>;IGNORE
-<NJ>	<n-j-sh>;<PCT>;<CAP>;IGNORE
+<U01CA>	<n-j-sh>;<PCT>;<CAP>;IGNORE
 <N-j>	<n-j-sh>;<PCT>;<CAP-MIN>;IGNORE
-<Nj>	<n-j-sh>;<PCT>;<CAP-MIN>;IGNORE
+<U01CB>	<n-j-sh>;<PCT>;<CAP-MIN>;IGNORE
 
 reorder-after <U0073>
 <U0161>	<scaron>;<PCT>;<MIN>;IGNORE
